src/pages/compare.tsx

```diff
@@ -31,7 +31,7 @@ export default ({ inputStr, results }: CompareProps) => {
             <PageContainer>
                 <SearchBar autoFocus={false} defaultValue={inputStr} />
                 <div style={{ maxWidth: '100%', overflow: 'auto' }}>
-                    <table style={{ marginTop: '30px' }}>
+                    <table className="compare" style={{ marginTop: '30px' }}>
```

Comment:
```suggestion
                    <table style={{ marginTop: '30px' }}>
```

Looks like this className is unused so it can be removed